src/languages/vbnet.js

```diff
@@ -33,30 +53,18 @@ export default function(hljs) {
     illegal: '//|{|}|endif|gosub|variant|wend|^\\$ ', /* reserved deprecated keywords */
     contains: [
       hljs.inherit(hljs.QUOTE_STRING_MODE, {contains: [{begin: '""'}]}),
-      hljs.COMMENT(
-        '\'',
-        '$',
-        {
-          returnBegin: true,
-          contains: [
-            {
-              className: 'doctag',
-              begin: '\'\'\'|<!--|-->',
-              contains: [hljs.PHRASAL_WORDS_MODE]
-            },
-            {
-              className: 'doctag',
-              begin: '</?', end: '>',
-              contains: [hljs.PHRASAL_WORDS_MODE]
-            }
-          ]
-        }
-      ),
+      commentMode,
       hljs.C_NUMBER_MODE,
       {
         className: 'meta',
-        begin: '#', end: '$',
-        keywords: {'meta-keyword': 'if else elseif end region externalsource'}
+        begin: '#(?:const|disable|else|elseif|enable|end|externalsource|if|region)\\b', end: '$',
+        keywords: {'meta-keyword': 'const disable else elseif enable end externalsource if region'},
+        contains: [commentMode]
+      },
+      {
+        className: 'literal',
+        begin: '#', end: '#',
```

Comment:
This is very broad, is there no format guidelines for the dates inside here?

Reply:
We likely need to handle a few specific date variants or reduce this to 0 relevancy (or both).  Some common formats could get 1 relevancy and then `#.*#` could default to 0 relevance if it does not look like a date.

Reply:
Right now something like `###########################` in markdown would count as 20 VB literals, which is not good at all.

Reply:
Done, added more specific regex.